app/assets/javascripts/short-form/components/preferenceComponent.js.coffee

```diff
@@ -52,7 +52,7 @@ angular.module('dahlia.components')
 
       # For 588 Mission preference, we need to override some defaults.
       @isEmploymentDisability = =>
-        @title == 'Employment/Disability Preference'
```

Comment:
This is a bit fragile. We updated the name in salesforce and the extra field just disappeared 😕 . Hopefully won't happen again